Makefile

```diff
@@ -53,5 +53,12 @@ pull-binaries-fabric:
 pull-binaries-fabric-ca:
 	./scripts/pullBinaries.sh latest fabric-ca
 
+.PHONY: caliper-init
+caliper-init:
+	cd $(CURDIR)/../caliper-benchmarks && npm init -y
+	npm install --only=prod @hyperledger/caliper-cli@0.3.1
+	npx caliper bind --caliper-bind-sut fabric:1.4.7
+	cd $(CURDIR)
+
```

Comment:
What is the `caliper-benchmarks` directory?

Also, I think it would suffice to reduce this logic to:

```
npm install -g --only=prod @hyperledger/caliper-cli@0.3.1
caliper bind --caliper-bind-sut fabric:1.4.7
```

This will install the version mentioned globally and it to the NPM path so you can call it directly using just `caliper` and not needing to use `npx`